.github/workflows/stale_prs.yml

```diff
@@ -14,5 +14,5 @@ jobs:
         stale-pr-message: "This PR has been inactive for long enough to be automatically marked as stale. This means it is at risk of being closed by a maintainer if it is not updated or reviews are not addressed. If your PR is closed as stale, feel free to open a new one after dealing with the issues. This may also be an indication that the maintainers do not have interest in this change, you can try to convince them otherwise, or persist in the doomed world you have created."
         stale-pr-label: 'Stale'
         exempt-pr-label: 'Needs Review'
-        days-before-stale: 30
+        days-before-stale: 100000
```

Comment:
Intended?

Reply:
We disabled auto stale.

Reply:
I never even enabled that workflow on here